pretalx_downstream/tasks.py

```diff
@@ -80,6 +101,12 @@ def process_frab(root, event, release_new_version):
     schedule = None
     if release_new_version:
         schedule_version = root.find("version").text
+
+        if event.settings.downstream_discard_after:
+            schedule_version = schedule_version.split(
+                event.settings.downstream_discard_after
+            )[0]
+
         try:
             event.wip_schedule.freeze(schedule_version, notify_speakers=False)
```

Comment:
Discarding parts of the incoming schedule version makes it more likely that the new version can't be imported, presumably, since versions need to be unique per event.

I'm not sure we need to fix that *now*, but we'll have to eventually.

Reply:
I do agree. However, i think this plugin is only used by a very small number of users, and the "trim schedule version name" feature gets used by even less people, that's why i think that risk is acceptable.

Maybe a future TODO could be to actually list (failed) imports in the admin ui, so it's easy to see wether something was broken.